src/vorta/application.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import os
 import sys
+from pathlib import Path, PurePath
```

Comment:
Why do you use PurePath instead of Path here?

Reply:
According to the [documentation](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module) this is best possible replacement of `os.path.join` and I think it works fine with `PurePath` as well. If there is any problem we can always use `Path.join`

Reply:
Yes, I think there is no real problem now with `PurePath`.

But you will also find this in the same doc page:
> If you’ve never used this module before or just aren’t sure which class is right for your task, [Path](https://docs.python.org/3/library/pathlib.html#pathlib.Path) is most likely what you need. It instantiates a [concrete path](https://docs.python.org/3/library/pathlib.html#concrete-paths) for the platform the code is running on.

I think it would be better if you would change `PurePath` to `Path`

Reply:
Done!